index.html

```diff
@@ -0,0 +1,211 @@
+                <p class="item item-content light-grey">
+                    Jsociety will be Hosting a coding challenge for attendees to test their skills and win prizes.
+                </p>
+            </article>
+        </div>
+        <button type="button" name="join-us" class="btn main-btn">
+            Join the Program
+        </button>
+        <a class="see-program"><u>SEE THE WHOLE PROGRAM</u></a>
+    </section>
+    <section class="featured-speakers flex-column-center" id="speakers">
+        <h2 class="featured-speakers-header light-black heading-medium">
+            Featured Speakers
+        </h2>
+        <hr class="horizontal-line text-center" />
+        <div class="speakers-list"></div>
+        <button type="button" name="join-us" class="btn speakers-btn" onclick="toggleSpeakers()">
+            MORE <i class="fas fa-angle-down down-arrow"></i>
+        </button>
+    </section>
+    <section class="partners bg-light-black flex-column-center" id="partners">
+        <h2 class="partners-heading light-grey heading-medium">Partners</h2>
+        <hr class="horizontal-line text-center" />
+        <ul class="partners-list flex-row-center">
+            <li class="list-item-container mozilla">
+                <img class="list-item" src="./Assets/images/tesla-motors.svg" alt="mozilla" />
+            </li>
+            <li class="list-item-container mozilla">
+                <img class="list-item" src="./Assets/images/firefox.svg" alt="mozilla" />
+            </li>
+            <li class="list-item-container mozilla">
+                <img class="list-item" src="./Assets/images/google-2015.svg" alt="mozilla" />
+            </li>
+            <li class="list-item-container mozilla">
+                <img class="list-item" src="./Assets/images/facebook-3.svg" alt="mozilla" />
+            </li>
+            <li class="list-item-container mozilla">
+                <img class="list-item" src="./Assets/images/amazon-lightsail.svg" alt="mozilla" />
+            </li>
+        </ul>
+    </section>
+    <footer class="footer flex-column-center" id="contact">
+        <h3 class="contact light-black heading-medium">Contact</h3>
+        <hr class="horizontal-line text-center" />
+        <div class="flex-row-center contact-details">
+            <img src="./Assets/images/Stem logo.png" alt="university of Nairobi logo"
+                class="uon-logo" />
+            <div class="adress">
+                <p class="phone-number"><b>Phone:</b>Tel: (+254-20) 12345667/ (+254-020) 2314566 <br>
+
+                    Fax: (+254-20) 1345674
+                </p>
+                <p class="adress-details">
+                    <b>Adress:</b> Javascript Society <br>
+                    P.O.Box 23132, GPO, Nairobi, Kenya <br>
+                    Twitter: @jsociety
+                </p>
+            </div>
+        </div>
+        <p class="rights"><i class="far fa-copyright"></i>2023. All rights reserved</p>
+    </footer>
```

Comment:
- [ ] Please remove the contact heading from the footer. Please make sure that the logo is aligned to the left and text aligned to the right and no more extra text at the bottom or top. Also pease reduce the padding of the footer section

Cindy's design | Your design
---|---
<img width="1038" alt="Screenshot 2023-07-07 at 11 58 36 AM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/2927ff4d-a2b7-4e7e-a309-a662156575ac">|<img width="1258" alt="Screenshot 2023-07-07 at 11 58 15 AM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/2fa95012-0c05-4ad2-a2c3-d938388b9a65">

- [ ] For About section footer desktop as well implement the above siggestions and please make sure that for About Page the background is dark with white colored text.

<img width="921" alt="Screenshot 2023-07-07 at 11 58 25 AM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/2ce11ab5-43c4-4c54-8f7a-714c0b222824">